src/Shared/Recruit.Vacancies.Client/Domain/Events/LiveVacancyClosingDateChangedEvent.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Esfa.Recruit.Vacancies.Client.Domain.Messaging;
+using MediatR;
+
+namespace Esfa.Recruit.Vacancies.Client.Domain.Events
+{
+    public class LiveVacancyClosingDateChangedEvent : EventBase, INotification
+    {
+        public Guid VacancyId { get; set; }
+        public long VacancyReference { get; set; }
+        public DateTime NewClosingDate { get; set; }
+
+        public LiveVacancyClosingDateChangedEvent() { }
```

Comment:
![Highlander Principle](https://media.giphy.com/media/9Jmb2idg10qJSygvTQ/giphy.gif)

Reply:
Typically I only create multiple ctor's on a class if the access modifier needs to be different.